tsl/test/expected/cagg_policy.out

```diff
@@ -1158,14 +1158,6 @@ ALTER MATERIALIZED VIEW mat_smallint SET (timescaledb.compress);
 NOTICE:  defaulting compress_orderby to a
 WARNING:  there was some uncertainty picking the default segment by for the hypertable: You do not have any indexes on columns that can be used for segment_by and thus we are not using segment_by for compression. Please make sure you are not missing any indexes
 NOTICE:  default segment by for hypertable "_materialized_hypertable_11" is set to ""
-\set ON_ERROR_STOP 0
-SELECT add_compression_policy('mat_smallint', 0::smallint);
-ERROR:  compress_after value for compression policy should be greater than the start of the refresh window of continuous aggregate policy for mat_smallint
-SELECT add_compression_policy('mat_smallint', -4::smallint);
-ERROR:  compress_after value for compression policy should be greater than the start of the refresh window of continuous aggregate policy for mat_smallint
-SELECT add_compression_policy('mat_bigint', 0::bigint);
-ERROR:  compress_after value for compression policy should be greater than the start of the refresh window of continuous aggregate policy for mat_bigint
-\set ON_ERROR_STOP 1
```

Comment:
I think you should leave the tests executing successfully to make evident that now it works and not remove them

Reply:
Okay, I remove had to change it to add and then remove because after this block we finally set the correct policy we want.